gotocompany/compass/v1beta1/service.proto

```diff
@@ -1146,6 +1178,58 @@ message DeleteAssetRequest {
 
 message DeleteAssetResponse {}
 
+message DeleteAssetsRequest {
+  string q = 1 [
+    (validate.rules).string.ignore_empty = true,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "filter by specific query"
+    }
+  ];
+  string q_fields = 2 [
+    (validate.rules).string.ignore_empty = true,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "filter by multiple query fields"
+    }
+  ];
+  string types = 3 [
+    (validate.rules).string.ignore_empty = true,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "filter by multiple types"
+    }
+  ];
+  string services = 4 [
+    (validate.rules).string.ignore_empty = true,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "filter by multiple services"
+    }
+  ];
+  map<string, string> data = 5 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+    description: "query result based on a (nested) field of the data asset. the nested field is written with period separated field name. eg, \"data[name.entity]\""
+  }];
+  string duration_for_upper_threshold = 6 [
```

Comment:
if we plan to use duration, let's use `google.protobuf.Duration` instead of `string`

Reply:
Okay, updated